resources/dicts/patrols/general/hunting.json

```diff
@@ -485,5 +485,31 @@
         "max_cats": 6,
         "antagonize_text": null,
         "antagonize_fail_text": null
-    }
+    },
+
+    {
+        "patrol_id": "gen_hunt_twolegdogsdeer1",
+        "biome": "Any",
+        "season": "newleaf",
+        "tags": ["hunting", "large_prey", "no_fail_prey", "warrior", "clan_to_patrol", "respect", "trust", "dislike"],
+        "intro_text": "The patrol hears a lot of barking and commotion.",
+        "decline_text": "p_l makes the decision to go to another part of the territory to hunt, they want no part of this ruckus.",
+        "chance_of_success": 25,
+        "exp": 10,
+        "success_text": [
+"The patrol silently creeps towards the noise. They discover several dogs gnawing on a deer carcass with their Twolegs shouting at them in apparent anger. The Twolegs grab the dogs and drag them away, their whines softly disappearing in the distance. p_l can't help but feel thankful to the dogs, c_n will eat well for several days to come.",
+"p_l and the patrol pause and snicker at some Twolegs' noisy attempt to catch the dogs gnawing on a deer carcass. As the Twolegs leave, the patrol begins their feast. A soft bleating noise notifies them of the presence of the deer's still nursing calf. They return to c_n with the calf's carcass, proud and tall."
+        ],
+        "fail_text": [
+"They wait till the Twolegs drag the dogs away and start to eat. Unfortunately it seems the dogs escaped their Twolegs grasp and the patrol has to run away. Both their bellies and that of their Clanmates will remain empty if no other hunting patrol has caught anything that day. They walk back to c_n in shame, dragging their tails."
```

Comment:
Reads a little confusingly.  You didn't introduce the idea of the deer carcass and dogs before saying that the twolegs drag the dogs away.  Away from what?  What's the patrol eating?  The next sentence also seems to be in past tense instead of present tense like the rest of the outcome, which makes it read oddly.

Reply:
You didn't fix the fail outcome I was talking about?